src/routes/components/AppFooter.tsx

```diff
@@ -141,46 +141,49 @@ export default function AppFooterPage() {
 
 
           {/* Examples*/}
-          <GoABlock gap="xs" direction="column" mt="2xl" mb="3xl">
-            <a href="#footer-meta">Footer with meta info</a>
-            <a href="#footer-secondary-nav">Secondary navigation column section and meta section</a>
-          </GoABlock>
+          <div className="component-example">
```

Comment:
Do we need this wrapping div. As far as I can see it just add some additional css selector specificity, but I don't think it is needed.